db/migrate/001_add_issues_project_key_and_number.rb

```diff
@@ -1,7 +1,7 @@
 class AddIssuesProjectKeyAndNumber < ActiveRecord::Migration
 
     def self.up
-        add_column :issues, :project_key,  :string, :limit => 10
+        add_column :issues, :project_key,  :string, :limit => 32
```

Comment:
It's not a good idea to modify migrate files, see more: http://stackoverflow.com/questions/1416600/rails-i-update-migration-file-then-run-dbmigrate-but-my-schema-isnt-updating
E.g. I have already run the migration, so if I run db migrate your changes will not be applied to my DB.
You should create a new migration for the changes. Otherwise I will not accept the pull request.
The same for other 3 changes in the migrate files.